### PR TITLE
Bump to Xcode 9 beta 4

### PR DESCRIPTION
**Pull request by spouliot**

### CODE REVIEW

**Comment by spouliot** [comment]
first build hung... and was killed

File: src/coreimage.cs [hunk 1]
```diff
@@ -1819,27 +1819,26 @@ interface CIWarpKernel {
...
 		[Static]
 		[Export ("imageAccumulatorWithExtent:format:")]
-#if !MONOMAC
-		[Internal]
-#endif
 		CIImageAccumulator FromRectangle (CGRect rect, int /* CIFormat = int */ ciImageFormat);
 
 		[iOS (9,0)]
-		[Static, Internal]
+		[Static]
```
Comment (dalexsoto) [inline src/coreimage.cs]: Why was this promoted? just curious, could `format` be an enum somewhere?

File: src/coreimage.cs [hunk 2]
```diff
@@ -1819,27 +1819,26 @@ interface CIWarpKernel {
...
 		
-
 		[Export ("initWithExtent:format:")]
-#if !MONOMAC
-		[Internal]
-#endif
 		IntPtr Constructor (CGRect rectangle, int /* CIFormat = int */ ciImageFormat);
 
-		[Export ("initWithExtent:format:colorSpace:")][Internal]
+		[Export ("initWithExtent:format:colorSpace:")]
```
Comment (dalexsoto) [inline src/coreimage.cs]: same, could `format` be an enum somewhere?

File: src/coreimage.cs (hunk 1 above)
Comment (spouliot) [inline src/coreimage.cs]: I have not _yet_ gone to the root of this :)
It's not clear why those methods were half hidden (inconsistent) in XM and not exposed at all (unusable) in XI.

**Comment by spouliot** [comment]
2nd time hung too (same bat place) on the same bot, trying another one...

**Comment by spouliot** [comment]
watchOS simulator issue filed in https://bugzilla.xamarin.com/show_bug.cgi?id=58348

File: src/coreimage.cs (hunk 1 above)
Comment (spouliot) [inline src/coreimage.cs]: https://bugzilla.xamarin.com/show_bug.cgi?id=58349

**Pull request merged**